nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -143,9 +144,10 @@ public ValidationResult validate(String subject, String input, ValidationContext
         }
     };
 
+    public static final String DELETE_ATTRIBUTES_EXPRESSION_NAME = "Delete Attributes Expression";
     // static properties
     public static final PropertyDescriptor DELETE_ATTRIBUTES = new PropertyDescriptor.Builder()
-            .name("Delete Attributes Expression")
+            .name(DELETE_ATTRIBUTES_EXPRESSION_NAME)
             .description("Regular expression for attributes to be deleted from FlowFiles.")
```

Comment:
I wanted to double check that the ordering of the delete vs update logic was preserved with what was documented but it doesn't appear to be at all. Your updates do seem to preserve the ordering as it was coded before (delete supersedes everything) but could you add a note either here or in the additional docs stating the intended order?